src/dymaptic.GeoBlazor.Core/Components/Layers/CSVLayer.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Layers;
+
+/// <summary>
+///     The CSVLayer is a point layer based on a CSV file (.csv, .txt). CSV is a plain-text file format used to
+///     represent tabular data, including geographic point features (latitude, longitude). Typically the latitude
+///     coordinate is the Y value, and the longitude coordinate is the X value. The X, Y coordinates must be stored
+///     in SpatialReference.WGS84 in csv feed.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-CSVLayer.html">
+///         ArcGIS
```

Comment:
These line breaks cause issues with our docs generation, I believe. We might need to inline this.

Reply:
ok, I think it was auto-formatted this way

Reply:
It looks like its formatted exactly like other layers in the project